python/tvm/relay/frontend/pytorch.py

```diff
@@ -1597,6 +1597,21 @@ def func(x):
 
         return func(data)
 
+    def var_mean(self, inputs, input_types):
+        data = inputs[0]
+        if len(inputs) == 2:
+            axis = None
+            keepdims = False
+            unbiased = bool(inputs[1])
+        else:
+            axis = inputs[1]
+            keepdims = bool(inputs[3])
+            unbiased = bool(inputs[2])
+
+        return _op.reduce.variance(data, axis=axis, keepdims=keepdims, unbiased=unbiased), _op.mean(
+            data, axis, keepdims
+        )
```

Comment:
See https://github.com/apache/tvm/blob/6df070aac6d0e26d1e127095a323c61c2287eb9d/python/tvm/relay/op/reduce.py#L356-L357

I think we can directly call `_make._variance` here to avoid computing `mean` twice.

Reply:
Thanks for your advice. Finally, I found an existing API `mean_variance`, and enhance it a bit.
https://github.com/apache/tvm/blob/6df070aac6d0e26d1e127095a323c61c2287eb9d/python/tvm/relay/op/reduce.py#L395